editor/src/clj/editor/workspace.clj

```diff
@@ -748,7 +748,8 @@ ordinary paths."
               (into #{}
                     (comp
                       (filter #(= :collision (:type %)))
-                      (mapcat :collisions))
+                      (mapcat :collisions)
+                      (filter #(re-matches #"^/[^/]*$" (key %))))
```

Comment:
Now we show collision notifications only for top-level folders since deps are always top-level.